tests/test_drawings.py

```diff
@@ -0,0 +1,118 @@
+        m.save(mission_path)
+
+        m2 = dcs.mission.Mission()
+        self.assertEqual(0, len(m2.load_file(mission_path)))
+        self.assert_expected_stuff(m2)
+        self.assertEqual("TEST CIRCLE", m2.drawings.layers[0].objects[1].name)
+        
+
+    def test_add_drawings_to_new_mission(self) -> None:
+        m: Mission = dcs.mission.Mission()
+        
+        circle = Circle(True, Point(10, 10), "TEST CIRCLE", Rgba(20, 30, 40, 200), ":S", Rgba(50, 60, 70, 150), 10, LineStyle.Solid, 100)
+        red_layer = m.drawings.get_layer(StandardLayer.Red)
+        red_layer.add_drawing(circle)
+        red_layer.add_line_segments(Point(1, 1), [Point(6,6), Point(7,7)], closed=True)
+
+        m.drawings.options.hiddenOnF10Map["Pilot"]["Red"] = True
+        m.drawings.options.hiddenOnF10Map["Instructor"]["Blue"] = True
+        m.drawings.options.hiddenOnF10Map["Observer"]["Neutral"] = True
+
+        mission_path = 'missions/New_mission_w_added_drawings.miz'
+        m.save(mission_path)
+
+        m2 = dcs.mission.Mission()
+        self.assertEqual(0, len(m2.load_file(mission_path)))
+        red_layer2 = m2.drawings.get_layer(StandardLayer.Red)
+        self.assertEqual("TEST CIRCLE", red_layer2.objects[0].name)
+        self.assertEqual("A line", red_layer2.objects[1].name)
+        self.assertEqual(True, red_layer2.objects[1].closed)
+        self.assertEqual("Red", red_layer2.objects[0].layer_name)
+        self.assertEqual("Red", red_layer2.objects[1].layer_name)
+
+
+    def test_set_options_hidden_f10(self) -> None:
+        m: Mission = dcs.mission.Mission()
+        
+        m.drawings.options.hiddenOnF10Map["Pilot"]["Red"] = True
+        m.drawings.options.hiddenOnF10Map["Instructor"]["Blue"] = True
+        m.drawings.options.hiddenOnF10Map["Observer"]["Neutral"] = True
+        mission_path = 'missions/New_mission_w_added_drawings.miz'
+        m.save(mission_path)
+
+        m2 = dcs.mission.Mission()
+        self.assertEqual(0, len(m2.load_file(mission_path)))
+        self.assertEqual(False, m2.drawings.options.hiddenOnF10Map["Pilot"]["Blue"])
+        self.assertEqual(True, m2.drawings.options.hiddenOnF10Map["Pilot"]["Red"])
+        self.assertEqual(True, m2.drawings.options.hiddenOnF10Map["Instructor"]["Blue"])
+        self.assertEqual(True, m2.drawings.options.hiddenOnF10Map["Observer"]["Neutral"])
+
+    def test_add_std_icon(self) -> None:
+        m: Mission = dcs.mission.Mission()
+
+        red_layer = m.drawings.get_layer(StandardLayer.Red)
+        red_layer.add_icon(Point(1000, 1000), StandardIcon.MechanizedArtillery)
+        mission_path = 'missions/New_mission_w_added_std_icon.miz'
+        m.save(mission_path)
+
+        m2 = dcs.mission.Mission()
+        self.assertEqual(0, len(m2.load_file(mission_path)))
+        red_layer = m.drawings.get_layer(StandardLayer.Red)
+        
+        self.assertEqual(StandardIcon.MechanizedArtillery.value, red_layer.objects[0].file)
```

Comment:
missing newline

Reply:
I would squash that commit, but it appears the merge of master I did into this branch would get lost/overwritten by picks if I did that... and I'm worried it will result in weirdness/conflicts when that is merged to master. 🤔 😬 

So I'd rather not, I think. Will fix the newline.